.github/workflows/pull_request_full_build.yml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - master
+      - 2201.[4-9]+.[0-9]+-stage
```

Comment:
Do we need this change since the fix being added to the `master` branch?

Reply:
I will remove this.